src/module-info.java

```diff
@@ -7,6 +7,11 @@
     requires aion.crypto;
     requires logback.core;
     requires logback.classic;
+    requires jdk.sctp;
 
     exports org.aion.api;
-}
\ No newline at end of file
+    exports org.aion.api.type;
+    exports org.aion.api.impl;
+    // should probably move what we need from this pkg into some pkg not named 'internal'
+    exports org.aion.api.impl.internal; 
```

Comment:
So, before this change, I think every package was being exported, because `module-info.java` was in aion_api's root folder instead of inside the src/ folder.  After moving it to src/, it is actually used by javac now, so compiling against it has become more restrictive.

I can work around the internal thing.  modGui looks at Message.java, which contains the mapping from string to numerical error code.  But I can just hardcode the numbers into modGui (or we can refactor those mappings to be put in a class/Enum in a package that is exported).

For org.aion.api.impl -- modGui is using it to get an instance of the Java API, through this method https://github.com/aionnetwork/aion_api/blob/master/src/org/aion/api/impl/AionAPIImpl.java#L60-L62.  That class is in org.aion.api.impl, which is why I had to export it.  Is there some other way to get an instance of the API?

The wallet will need it too: https://github.com/aionnetwork/Desktop-Wallet/blob/master/src/main/java/org/aion/wallet/connector/api/ApiBlockchainConnector.java#L43